modules/nf-core/gtdbtk/classifywf/meta.yml

```diff
@@ -30,6 +30,9 @@ input:
       type: file
       description: The local copy of the taxonomic database used by GTDB-tk (unzipped copy)
       pattern: "*"
+  - use_pplacer_scratch_dir:
```

Comment:
Do we need this at all?

Reply:
Yes we were need the switch to know whether to create the directory